src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -11,6 +12,8 @@
  */
 public class Tag {
 
+    public static final String[] BANNED_TAG_NAMES = new String[]{"create", "c", "remove", "r"};
```

Comment:
Should we define `BANNED_TAG_NAMES` in `TagCommandGroup` instead? To keep coupling low.

Reply:
Good point!